codegen/tests/example.rs

```diff
@@ -1,12 +1,12 @@
 use std::{convert::Infallible, time::Duration};
 
 use async_trait::async_trait;
-use cucumber_rust::{given, StepContext, World, WorldInit};
+use cucumber_rust::{gherkin::Step, given, World, WorldInit, WorldRun as _};
```

Comment:
It's better to re-shake traits here in the way we don't need additional `WorldRun as _` import, as this is unpleasant ergonomics hit.